src/main/java/io/vertx/codegen/type/AnnotationValueTypeInfo.java

```diff
@@ -0,0 +1,257 @@
+package io.vertx.codegen.type;
+
+import java.util.List;
+
+/**
+ * @author <a href="mailto:cafeinoman@openaliasbox.org>Francois Delalleau</a>
+ */
+public class AnnotationValueTypeInfo {
```

Comment:
I'm not sure why this class is needed and why instead we don't simply use a Map<String, Object> to represents the value of an annotation.